chicago/bills.py

```diff
@@ -86,7 +91,7 @@ def parseSearchResults(self, page) :
 
     def scrape(self):
 
-        for i, page in enumerate(self.searchLegislation()) :
+        for page in self.searchLegislation(created_after=datetime.datetime(2014, 1, 1), created_before=datetime.datetime(2014, 2, 1)) :
```

Comment:
Got date range filtering working again. Just putting this in now, not sure the best way to really pass this through. Probably through command line argument?